vault_program/src/burn_withdrawal_ticket.rs

```diff
@@ -76,7 +76,16 @@ pub fn process_burn_withdrawal_ticket(
         fee_amount,
         burn_amount,
         out_amount,
-    } = vault.burn_with_fee(vault_staker_withdrawal_ticket.vrt_amount(), min_amount_out)?;
+    } = vault.burn_with_fee(
+        vault_staker_withdrawal_ticket.vrt_amount(),
+        vault_staker_withdrawal_ticket.min_amount_out(),
+    )?;
+
+    if out_amount < vault_staker_withdrawal_ticket.min_amount_out() {
+        msg!("Min amount out is not met");
+        return Err(VaultError::SlippageError.into());
```

Comment:
can you test this condition?

Reply:
10-4

Reply:
It turns out, I cannot test this through normal means. 

What I tried:
- enqueue_withdrawal
- slash [ Fails - slash amount exceeds total security, because of the enqueue_withdrawal ]
- burn_withdrawal_ticket

Are there other means of decreasing the token_deposited?

Reply:
Never mind, test is working